README.md

```diff
@@ -15,12 +15,12 @@ Refer to our Meetup page: https://www.meetup.com/Kalispell-Software-Crafters/
 - [ ] Add GitHub Section
 - [ ] Add Slack and Discord Section
 - [ ] Add Initial Basic Styles
```

Comment:
Thanks for making the changes I asked for! Would you also mark `Add Initial Basic Styles` complete with an X mark and add the spaces back in front of the sub-tasks below it? I see it looks like you removed those spaces. I also see now that was a little confusing but the tasks you're completing are actually sub-tasks of `Add Initial Basic Styles`. So once this is merged in that will be completed. 😄 

I appreciate the help with all this! 

I also plan to close out the extra PRs you've submitted since we want to keep all the changes in one PR.